### ISSUE TITLE
Page nos in bib refs

### ISSUE THREAD

**Issue by AndrewSales** [sgmlguru/MUK-proc#4]
Worth turning off? Can look a bit busy, see esp. Lafontaine.

https://github.com/sgmlguru/MUK-proc/blob/6367f1b0176f8e8dcab9e78045f79da93a8c7633/xsl/muk-proceedings-db.xsl#L46

**Comment by sgmlguru** [comment]
No, I think they are needed. I can't stand xrefs without proper means to find the target.

**Comment by AndrewSales** [comment]
Even when they're always at the end of the article? I'll defer to you.

**Comment by AndrewSales** [comment]
See Graham - not sure if this the cause:

![image](https://user-images.githubusercontent.com/1850015/41129271-3d6032be-6aa9-11e8-8b26-f9a16c8b9d94.png)

**Comment by AndrewSales** [comment]
Ignore me: it's cos he had the ID on the title, not the section.

**Comment by tgraham-antenna** [comment]
FWIW, I agree with @AndrewSales. Sections and figures/tables are numbered, and the bibliography is only in one place.

Bunce (2019) manages to avoid page number references:

https://github.com/sgmlguru/MUK-proc/blob/7d9c13451ab860161c924b8cacc1f38db0b3d753/papers-2019/andy/paper.xml#L88
